wbooks/src/screens/home/layout.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { SELECT_FILTER, TITLE, AUTHOR, SEARCH } from './strings';
+import styles from './styles.scss';
+import propTypes from '../../consts/propTypes';
+import BookCardContainer from '../../components/book-card/index';
+import SearchImage from '../../images/search.svg';
+
+class Home extends Component {
+  render() {
+    return (
+      <div className={styles.container}>
+        <div className={styles.form}>
+          <select name='type' className={classNames({[styles.select]: true, [styles.input]: true})} onChange={this.props.handleTypeChange}>
+            <option className={styles.option} value='' default>{SELECT_FILTER}</option>
+            <option className={styles.option} key='title' value='title'>{TITLE}</option>
+            <option className={styles.option} value='author'>{AUTHOR}</option>
```

Comment:
I'd extract `title`, `author` into `consts`. Does they relate to the other ones ?